wework/src/styles/globals.css

```
@@ -672,6 +701,18 @@ button:disabled {
     object-fit: contain;
   }
 
+  .composer-mention-initial-icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border-radius: 0.2em;
+    background: rgb(var(--color-focus) / 0.1);
+    color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)